app/views/users/new.html.erb

```diff
@@ -15,6 +15,11 @@
     </div>
 
     <div class='form-group'>
+      <%= f.label :location, class: 'col-sm-2 control-label' %>
```

Comment:
'f' is left ambiguous in this version. Do we want to use 'formElement' as Ardeshir's latest pull request suggests?